docs/user/checks.rst

```diff
@@ -922,6 +922,23 @@ Vue I18n formatting
    `Vue I18n Formatting <https://kazupon.github.io/vue-i18n/guide/formatting.html>`_,
    `Vue I18n Linked locale messages <https://kazupon.github.io/vue-i18n/guide/messages.html#linked-locale-messages>`_
 
+.. _check-automattic-components-format:
+
+Automattic components formatting
+********************************
+
+:Summary: The Automattic components' placeholders do not match the source
+:Scope: translated strings
+:Check class: ``weblate.checks.format.AutomatticComponentsCheck``
+:Check identifier: ``automattic_components_format``
+:Flag to enable: ``automattic-components-format``
+:Flag to ignore: ``ignore-automattic-components-format``
+:Simple format string example: ``They bought {{strong}}apples{{/strong}}.``
+
+.. seealso::
+
+   :ref:`check-formats`
```

Comment:
Can you please also include links to the upstream documentation?

Reply:
I tried to find links to the upstream documentation but didn't find it. I'll ask in the original issue.

Reply:
I've found https://github.com/Automattic/wp-calypso/tree/trunk/packages/interpolate-components, not full docs, but still better than nothing.

Reply:
Done.